### PR TITLE
Adding Spanish translation

### PR DESCRIPTION
**Pull request by carlogarro**
I added most of the translation.
I will finish within the next day the left ones.

### CODE REVIEW

File: po/es_AR.po [hunk 1]
```diff
@@ -101,6 +101,14 @@ msgid ""
 "the world. You can also search for the white rabbit on <b>http://world.kano."
 "me</b>"
 msgstr ""
+"En 1978, el ordenador ms rápido del mundo fue el Cray 1 - costaba $ 7 "
```
Comment (skarbat) [inline po/es_AR.po]: Falta la a en`más`

File: po/es_AR.po [hunk 2]
```diff
@@ -101,6 +101,14 @@ msgid ""
...
+"millones y pesaba tanto como un elefante! Tu Kano es aproximadamente 4 "
```
Comment (skarbat) [inline po/es_AR.po]: Sugiero usar la forma `7 millones de dólares`.

File: po/es_AR.po [hunk 3]
```diff
@@ -101,6 +101,14 @@ msgid ""
...
+"veces más rápida que el Cray 1. Hablando de velocidad, vamos a descargar nuevos "
+"juegos, apps y proyectos, organizándolo en la memoria del ordenador. Esto "
+"debera tardar unos 15 minutos. Mientras tanto, que tal si le haces una foto "
```
Comment (skarbat) [inline po/es_AR.po]: `debería`

File: po/es_AR.po [hunk 4]
```diff
@@ -169,257 +177,265 @@ msgstr ""
...
 msgid "Almost there..."
-msgstr ""
+msgstr "Ya casi estamos..."
 
 #: ../kano_updater/ui/available_window.py:128
 msgid ""
 "Downloaded and ready to go! In a matter\n"
 "of minutes your Kano will be fresher than ever"
 msgstr ""
+"Descargado y listo para empezar! En cuestón\n"
```
Comment (skarbat) [inline po/es_AR.po]: `cuestión`

File: po/es_AR.po [hunk 5]
```diff
@@ -169,257 +177,265 @@ msgstr ""
...
 msgid "Best time to update is now!"
-msgstr ""
+msgstr "El mejor momento para actualizar es ahora!"
 
 #: ../kano_updater/ui/available_window.py:146
 msgid ""
 "Latest updates and content are ready to go! We\n"
 "will make sure to Shutdown once we finish installing!"
 msgstr ""
+"Actualizaciones y recientes y contenido están listos! Vamos \n"
```
Comment (skarbat) [inline po/es_AR.po]: Sobra la primera `y`.

File: po/es_AR.po [hunk 6]
```diff
@@ -521,97 +545,101 @@ msgid ""
 "\n"
 "Python modules upgraded:\n"
 msgstr ""
+"\n"
+"Módulos Pythos actualizados:\n"
```
Comment (skarbat) [inline po/es_AR.po]: `Python`

File: po/es_AR.po [hunk 7]
```diff
@@ -521,97 +545,101 @@ msgid ""
...
 
 #: ../kano_updater/dialogs.py:76
 msgid ""
 "\n"
 "Python modules with error:\n"
 msgstr ""
+"\n"
+"Python modules con errores:\n"
```
Comment (skarbat) [inline po/es_AR.po]: `Módulos Python con errores:\n`

**Comment by carlogarro** [comment]
I have fixed everything that skarbat told me. Thanks for reading it carefuly!
Now, the document is finished, so we need a last review and then we can submit it!

Gracias!

**Comment by skarbat** [comment]
Looks good Carlos @carlogarro - thanks! Merging right away

**Pull request merged**